doc/source/getting_started/learning.rst

```diff
@@ -95,7 +95,7 @@ Course content
 * Post Completion Survey
 * Meet the Instructors
 
-Ansys Innovation Space
+Ansys innovation space
```

Comment:
I'd suggest leaving the initial capital letters in this case because "Ansys Innovation Space" is a proper name. You can tun Vale off before this title and back on.

Reply:
Roger that! :)